providers/ms365/resources/ms365.lr

```diff
@@ -95,6 +97,156 @@ microsoft.conditionalAccess.namedLocations {
   countryLocations() []microsoft.conditionalAccess.countryNamedLocation
 }
 
+ // Container for Microsoft Conditional Access Policies (Collection Resource)
+microsoft.conditionalAccess.policies {
+  []microsoft.conditionalAccess.policy
+  // init(filter? string, search? string)  
+  // filter string
+  // search string
+}
+
+// Microsoft Conditional Access Policy
+microsoft.conditionalAccess.policy @defaults("id displayName state") {
+  // Policy ID
+  id string
+  // Policy display name
+  displayName string
+  // Policy state (enabled, disabled, enabledForReportingButNotEnforced)
+  state string
+  // Policy creation date
+  createdDateTime time
+  // Policy last modified date
+  modifiedDateTime time
+  // Policy conditions
+  conditions microsoft.conditionalAccess.policy.conditions // Now a specific resource
+  // Policy grant controls
+  grantControls microsoft.conditionalAccess.policy.grantControls // Now a specific resource
+  // Policy session controls
+  sessionControls microsoft.conditionalAccess.policy.sessionControls // Now a specific resource
+  // Template ID (if created from template)
+  templateId string
+}
+
+// Microsoft Conditional Access Policy Conditions
+private microsoft.conditionalAccess.policy.conditions {
+  // Internal ID based on policy ID
+  id string
+  // Users affected by the policy
+  users microsoft.conditionalAccess.policy.conditions.users
+  // Applications affected by the policy
+  applications microsoft.conditionalAccess.policy.conditions.applications
+  // Client app types affected by the policy
+  clientAppTypes []string
+  // User risk levels affected by the policy
+  userRiskLevels []string
+  // Sign-in risk levels affected by the policy
+  signInRiskLevels []string
+  // service principal risk levels affected by the policy
+  servicePrincipalRiskLevels []string
+  // Locations affected by the policy
+  locations microsoft.conditionalAccess.policy.conditions.locations
+  // Device platforms affected by the policy
+  platforms microsoft.conditionalAccess.policy.conditions.platforms
+  // Client applications affected by the policy
+  clientApplications microsoft.conditionalAccess.policy.conditions.clientApplications
+  // insider risk levels
+  insiderRiskLevels string
+}
+
+private microsoft.conditionalAccess.policy.grantControls.authenticationStrength {
```

Comment:
Even though these are private they'll need the docs comments since they all end up in the generated docs